src/core/lambda/io.rs

```diff
@@ -27,6 +27,7 @@ pub enum IO {
         req_template: http::RequestTemplate,
         group_by: Option<GroupBy>,
         dl_id: Option<DataLoaderId>,
+        http_filter: HttpFilter,
```

Comment:
This could be optional. If not specified, we should not unnecessarily make a round trip to the JS runtime.

Reply:
if we dont have/want ant filter then we pass HttpFilter::default(). WE have a check in request Filter logic to call onRequest or execute directly.